content/applications/productivity/discuss/advanced/ICE_servers.rst

```diff
@@ -0,0 +1,71 @@
+========================================
+Configure ICE servers with Twilio
+========================================
+
+Odoo Discuss uses WebRTC API and peer-to-peer connections for voice and video calls. If one of the
+call attendees is behind a symmetric NAT, you need to configure an ICE server to establish a
+connection to the call attendee. To do that, you must create a Twilio account for video calls, and
+then, connect your Twilio account to Odoo.
```

Comment:
I try to avoid _this_ and _that_, where possible, to enforce clarity of language.

In this case a lot of technical terms were mentioned in a short section of writing. The most important piece of information for the user here is that they need an ICE server, so it's worth repeating again in the following sentence alongside the solution provider, Twilio.

Also removed 2nd person imperative.

```suggestion
connection to the call attendee. To set up an ICE server, create a Twilio account for
video calls, and then connect that Twilio account to Odoo.
```